src/main/java/ua/cv/tim/service/impl/UserServiceImpl.java

```diff
@@ -148,14 +150,14 @@ public void addUser(UserDTO member) throws MessagingException {
 		player.setAlliance(allianceDao.getAllianceByName(member.getAlliance()));
 		user.setPlayer(player);
 		playerDao.add(player);
-		sendEmail(user, password);
+		sendEmail(user,"Your login is \"" + user.getLogin() + "\", and password: \""
```

Comment:
message created inside method before modification, but now we need to modify message for create and update methods, so  decision to modify message text in params was taken and method was modified